examples/http/client.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+require 'rubygems'
+require 'bundler/setup'
+require 'faraday'
+# Require otel-ruby
+require 'opentelemetry'
+require 'opentelemetry/sdk'
+
+require_relative '../exporters/console'
+
+# Allow setting the host from the ENV
+host = ENV.fetch('HTTP_EXAMPLE_HOST', '0.0.0.0')
+
+SDK = OpenTelemetry::SDK
+OpenTelemetry.tracer_factory = SDK::Trace::TracerFactory.new
```

Comment:
This is a little annoying - we should really auto-register the SDK pieces if you've `require`d them.